<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserFinderSettingsGUI</class>
 <widget class="QDialog" name="UserFinderSettingsGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>505</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>User Finder Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Images/settings_single_gear.png</normaloff>../Images/settings_single_gear.png</iconset>
  </property>
  <property name="whatsThis">
   <string>Settings for the user finder</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="autorun_group_box">
     <property name="title">
      <string>Auto-Run Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="run_with_main_checkbox">
        <property name="text">
         <string>Run With Main Downloader</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="auto_add_checkbox">
        <property name="text">
         <string>Auto Add Found Users</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="silent_run_checkbox">
        <property name="text">
         <string>Auto Run Silently</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Auto Add User List:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QComboBox" name="auto_add_user_list_combo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="content_double_click_group_box">
     <property name="title">
      <string>Content Double Click Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="open_link_in_dialog_radio">
        <property name="text">
         <string>Open Link In Dialog</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="open_link_in_browser_radio">
        <property name="text">
         <string>Open Link In Browser</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="found_user_options_group_box">
     <property name="title">
      <string>Found User Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="show_reddit_page_checkbox">
        <property name="toolTip">
         <string>When selected the users reddit page will be displayed instead of samples of links</string>
        </property>
        <property name="text">
         <string>Show Users Reddit Page</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QSpinBox" name="sample_size_spinbox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Number of Samples</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="user_new_radio">
        <property name="toolTip">
         <string>Displayed posts will be sorted by new</string>
        </property>
        <property name="text">
         <string>Users Newest Posts</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="user_top_radio">
        <property name="toolTip">
         <string>Displayed posts will be sorted by top</string>
        </property>
        <property name="text">
         <string>Users Top Posts</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QRadioButton" name="user_selected_new_radio">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When selected the posts that earned the user a spot in the found users list will be displayed first followed by the selected number of the usrers new posts.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Selected Posts
Followed By New</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QRadioButton" name="user_selected_only_radio">
        <property name="toolTip">
         <string>Will provide samples of only the posts that were in the top for the selected subreddits.  Sample size will be ignored.</string>
        </property>
        <property name="text">
         <string>Selected Posts Only</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QRadioButton" name="user_selected_top_radio">
        <property name="toolTip">
         <string>When selected the posts that earned the user a spot in the found users list will be displayed first followed by the selected number of the usrers top posts.</string>
        </property>
        <property name="text">
         <string>Selected Posts
Followed By Top</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" rowspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
